### PR TITLE
Fix eslint error & Revert event file changes

### PR DESCRIPTION
**Pull request by Imamah-Zafar**
## Description

- Fix eslint issues 
- Revert changes in event.ts file due to git rebase

### CODE REVIEW

**Review by fariedt — APPROVED** [review]

**Review by fariedt — APPROVED** [review]

File: backend/backend/backend-src/routes/events-api.ts [hunk 1]
```diff
@@ -2,44 +2,49 @@
...
+      await handleGetPackageEvents(req, res);
+    }
+    catch (error) {
+      next(error);
     }
-  })
+  });
 
-  router.get('/:address', async (req, res, next) => {
+  router.get('/address/:address', async (req, res, next) => {
```
Comment (asimm241) [inline backend/backend/backend-src/routes/events-api.ts:30]: @Imamah-Zafar shouldn't this be done in separate PR as it is not related to ESlint

**Review by asimm241 — APPROVED** [review]
